### PR TITLE
Added jquery.rest lib 1.0.2

### PR DESCRIPTION
**Pull request by lievenjanssen**
github: https://github.com/jpillora/jquery.rest
release: https://github.com/jpillora/jquery.rest/archive/1.0.2.zip

### CODE REVIEW

File: ajax/libs/jquery.rest/1.0.2/jquery.rest.min.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// jQuery REST Client - v0.0.7 - https://github.com/jpillora/jquery.rest
```
Comment (sutra) [inline ajax/libs/jquery.rest/1.0.2/jquery.rest.min.js]: The version of the file content(v0.0.7) is not the declared version(v1.0.2).

**Comment by PeterDaveHello** [comment]
@treshugart  see https://github.com/jpillora/jquery.rest/blob/1.0.2/dist/jquery.rest.js, https://github.com/jpillora/jquery.rest/blob/1.0.2/dist/jquery.rest.min.js

**Comment by sutra** [comment]
> @sutra see https://github.com/jpillora/jquery.rest/blob/1.0.2/dist/jquery.rest.js, https://github.com/jpillora/jquery.rest/blob/1.0.2/dist/jquery.rest.min.js

Check the subdirectory `1`, so the files `dist/*.js` are `0.x`, and the `dist/1/*.js` are `1.x`:
https://github.com/jpillora/jquery.rest/blob/1.0.2/dist/1/jquery.rest.js, https://github.com/jpillora/jquery.rest/blob/1.0.2/dist/1/jquery.rest.min.js

**Comment by PeterDaveHello** [comment]
Interesting ... I'll take a look inside.

**Comment by PeterDaveHello** [comment]
@terinjokes would you please help us flush the cache of `https://cdnjs.cloudflare.com/ajax/libs/jquery.rest/1.0.2/jquery.rest.min.js`? Thanks.

**Pull request merged**